packages/sw-background-sync-queue/demo/index.html

```diff
@@ -0,0 +1,28 @@
+<html>
+	<head></head>
+	<body>
+		<div>Hello dear</div>
+		<button onclick="putReqInQ('1')">send req</button>
+		<button onclick="putReqInQ('2')">send req</button>
+		<button onclick="putReqInQ('3')">send req</button>
+		<button onclick="putReqInQ('4')">send req</button>
+		<button onclick="putReqInQ('5')">send req</button>
+		<script>
```

Comment:
please omit everything in demo, as I'll be building two different demos with and without custom fetch handler.
this was just to see if the POC was working